packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -14,7 +14,7 @@ export default class Hello extends Component {
 `;
 
 exports[`fix > .gts > when missing a local prop 1`] = `
-"import Component from \\"@glimmer/component\\";
```

Comment:
This will be addressed next PR

Reply:
Oh this is a prettier config issue where it's not recognizing the correct config because we are passing `packageDir` to the plugins?

For each case of a plugin, should be findingup to find .eslintrc or .prettierrc?